### ISSUE TITLE
Introduction - subject-verb misalignment

### ISSUE THREAD

**Issue by msellon** [w3c/silver#292]
There is a mix of verb forms in the final paragraph of the 1. Introduction intro section, which should be aligned:

1st Misalignment:
>The Web Content Accessibility Guidelines (WCAG) 2.0 were designed to be technology neutral, and has stayed relevant for over 10 years.

As 'Guidelines' is plural, 'has' should become 'have':
The Web Content Accessibility Guidelines (WCAG) 2.0 were designed to be technology neutral, and **have** stayed relevant for over 10 years.

2nd Misalignment:
>The Authoring Tool Accessibility Guidelines (ATAG) 2.0 has been implemented in the open source authoring tool communities (chiefly Wordpress and Drupal) with little known uptake in commercial authoring tools. 

As 'Guidelines' is plural, 'has' should become 'have':
The Authoring Tool Accessibility Guidelines (ATAG) 2.0 **have** been implemented in the open source authoring tool communities (chiefly Wordpress and Drupal) with little known uptake in commercial authoring tools.

**Comment by lauracarlson** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by jspellman** [comment]
While the word "guidelines" is plural, the document is singular.  I mess it up all the time unless I am focusing very hard on using the singular version.  The [EO manual of style document wants it singular](https://www.w3.org/WAI/EO/wiki/Style#Referring_to_xxAG_.28WCAG_2.3B_singular.29).  But regardless, it needs to be fixed because it is mixed.

**Comment by jspellman** [comment]
PR #619

**Comment by alastc** [comment]
Closed by #619
